### ISSUE TITLE
Could not pass event InventoryClickEvent to ItemJoin

### ISSUE THREAD

**Issue by mibby** [RockinChaos/ItemJoin#315]
```
[08:29:26] [Server thread/ERROR]: Could not pass event InventoryClickEvent to ItemJoin v5.0.7-SNAPSHOT-b475
java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 36
	at java.util.Arrays$ArrayList.get(Arrays.java:4164) ~[?:?]
	at net.minecraft.server.v1_16_R1.NonNullList.get(SourceFile:46) ~[patched_1.16.1.jar:git-Tuinity-"f265b38"]
	at net.minecraft.server.v1_16_R1.PlayerInventory.getItem(PlayerInventory.java:541) ~[patched_1.16.1.jar:git-Tuinity-"f265b38"]
	at org.bukkit.craftbukkit.v1_16_R1.inventory.CraftInventory.getItem(CraftInventory.java:49) ~[patched_1.16.1.jar:git-Tuinity-"f265b38"]
	at me.RockinChaos.itemjoin.listeners.Interact.onEquipClickCommand(Interact.java:134) ~[?:?]
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[patched_1.16.1.jar:git-Tuinity-"f265b38"]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[patched_1.16.1.jar:git-Tuinity-"f265b38"]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.1.jar:git-Tuinity-"f265b38"]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.16.1.jar:git-Tuinity-"f265b38"]
	at net.minecraft.server.v1_16_R1.PlayerConnection.a(PlayerConnection.java:2438) ~[patched_1.16.1.jar:git-Tuinity-"f265b38"]
	at net.minecraft.server.v1_16_R1.PacketPlayInWindowClick.a(SourceFile:32) ~[patched_1.16.1.jar:git-Tuinity-"f265b38"]
	at net.minecraft.server.v1_16_R1.PacketPlayInWindowClick.a(SourceFile:10) ~[patched_1.16.1.jar:git-Tuinity-"f265b38"]
	at net.minecraft.server.v1_16_R1.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:43) ~[patched_1.16.1.jar:git-Tuinity-"f265b38"]
	at net.minecraft.server.v1_16_R1.TickTask.run(SourceFile:18) ~[patched_1.16.1.jar:git-Tuinity-"f265b38"]
	at net.minecraft.server.v1_16_R1.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.16.1.jar:git-Tuinity-"f265b38"]
	at net.minecraft.server.v1_16_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.16.1.jar:git-Tuinity-"f265b38"]
	at net.minecraft.server.v1_16_R1.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.16.1.jar:git-Tuinity-"f265b38"]
	at net.minecraft.server.v1_16_R1.MinecraftServer.aZ(MinecraftServer.java:1191) ~[patched_1.16.1.jar:git-Tuinity-"f265b38"]
	at net.minecraft.server.v1_16_R1.MinecraftServer.executeNext(MinecraftServer.java:1184) ~[patched_1.16.1.jar:git-Tuinity-"f265b38"]
	at net.minecraft.server.v1_16_R1.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.16.1.jar:git-Tuinity-"f265b38"]
	at net.minecraft.server.v1_16_R1.MinecraftServer.sleepForTick(MinecraftServer.java:1160) ~[patched_1.16.1.jar:git-Tuinity-"f265b38"]
	at net.minecraft.server.v1_16_R1.MinecraftServer.v(MinecraftServer.java:1004) ~[patched_1.16.1.jar:git-Tuinity-"f265b38"]
	at net.minecraft.server.v1_16_R1.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.1.jar:git-Tuinity-"f265b38"]
	at java.lang.Thread.run(Thread.java:832) [?:?]
```

**To Reproduce**
Not entirely sure, noticed it in my console logs when scrubbing for problems.

**Expected behavior**
No error?

**Please complete the following information:**
ItemJoin dev 475
Tuinity dev 317 (Paper Spigot 1.16.1)

**Comment by RockinChaos** [comment]
Should now be fixed in the latest snapshot; http://ci.craftationgaming.com/job/ItemJoin/478/
Let me know!

**Comment by RockinChaos** [comment]
This bug has been officially fixed in the release of ItemJoin v5.0.7.
You can grab it here; Download: https://www.spigotmc.org/resources/itemjoin.12661/download?version=354306

Please read the changelog as the documentation has not been updated yet;
https://github.com/RockinChaos/ItemJoin/wiki/Recent-Changes
